etc/bootstrap-example/README.md

```diff
@@ -0,0 +1,23 @@
+# Bootstrap Example
+
+Assuming you are in MO's source directory.
+
+Build mo-service -
+```
+make service
+```
+Start 3 mo-service instances configured as Log Service instances in 3 different terminals on the same machine. 
+
+```
+./mo-service -cfg etc/log-node-1.toml
```

Comment:
```
./mo-service -cfg etc/bootstrap-example/log-node-1.toml
./mo-service -cfg etc/bootstrap-example/log-node-2.toml
./mo-service -cfg etc/bootstrap-example/log-node-3.toml
```

Reply:
It may like this: ` ./mo-server ./etc/bootstrap-example/log-node-1.toml`

Reply:
fixed